bin/cylc-cycle-point

```diff
@@ -252,15 +252,15 @@ def main():
             parser.error('ERROR: offset not valid: %s' % exc)
     cycle_point += offset
     if template is None:
-        print cycle_point
+        print(cycle_point)
     else:
         dump_string = iso_point_dumper.dump(cycle_point, template)
         if (dump_string == template and
                 not any(_ in template for _ in ["CCYY", "MM", "DD", "%"])):
             # A pure time string, no date - not well handled in isodatetime.
-            print iso_point_dumper.dump(cycle_point, "T" + template)[1:]
+            print(iso_point_dumper.dump(cycle_point, "T" + template)[1:])
         else:
-            print dump_string
+            print(dump_string)
 
 
 if __name__ == "__main__":
```

Comment:
Need to figure out how to make this a simple wrapper of `isodatetime`. @wxtim is on the case for `rose date`.